build/root/Makefile

```diff
@@ -179,6 +179,8 @@ check test:
 	@echo "$$CHECK_TEST_HELP_INFO"
 else
 check test: generated_files
+	$(eval UNKNOWN = $(shell echo $(MAKEFLAGS) | sed s/"\s"/"\n"/g | grep "=" | grep -v "^WHAT=" | grep -v "^TESTS=" | grep -v "^KUBE_COVER=" | grep -v "^GOFLAGS=" | grep -v "^GOLDFLAGS=" | grep -v "^GOGCFLAGS=" | grep -v "^KUBE_TEST_ARGS=" | grep -v "^KUBE_TEST_API_VERSIONS=" | grep -v "^KUBE_TIMEOUT=" | grep -v "^KUBE_RACE=" | grep -v "^--vmodule=" | grep -v "^--alsologtostderr="))
```

Comment:
this seems somewhat unwieldy; it also still ignores a number of options, since most of these (except for WHAT and TESTS) are environment variables that override settings in the `hack/make-rules/test.sh` script. The list of args in the Makefile help is not up to date, and neither is this list.

this is basically a core problem of using environment variables for configuration: there's really no way to adequately validate them.

Reply:
@ixdy Thanks for your comment. Yeah I felt the same thinking when adding environment variable names on this excluded list of Makefile, because the names are more many than I expected.

We can update the list of args help as you said, but the issue report itself could not be solved.
I cannot close the issue because of the permission, could you do that for avoiding the same work in the future?

Reply:
@xialonglee No problem :) OK, let's close this also